federated_learning/nvflare/nvflare_example/fl_workspace/admin/startup/client.crt

```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
+MIICtDCCAZygAwIBAgIUDIiaFScPM7kJ+y6TLJIZgIQPW1UwDQYJKoZIhvcNAQEL
```

Comment:
Should the files that can be produced by the provision tool be removed in the original folder? If users need to changed the `project.yml` and/or `authz_config.json`, all these files are useless.

I suggest to keep the style similar for both docker and non-docker examples, thus users may not need to spend extra times learning these two different ways.

Reply:
Thanks. This tutorial is meant to make the execution easier by skipping the provisioning step and providing ready-made startup kits to run the server and clients on localhost. I added some information about this in the readmes.